auxiliary/ArbitrumGasPriceOracle/contracts/interfaces/ArbGasInfo.sol

```diff
@@ -0,0 +1,152 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity >=0.4.21 <0.9.0;
+
+/// @title Provides insight into the cost of using the chain.
```

Comment:
could you add as comment where this was pulled from. Thanks

Reply:
done ✔️